extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.css

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-story-dev-tools-tab-logs {
+  overflow-y: scroll !important;
+  margin-top: 30px !important;
+  scrollbar-width: none !important;
+  --i-amphtml-story-dev-tools-logs-error-color: #FF5252 !important;
+  --i-amphtml-story-dev-tools-log-message-background: #ffffff0f;
+  --i-amphtml-story-dev-tools-log-spec-white-accent: #ffffff22;
```

Comment:
Re: colors, please find something so we can re-use them and don't end up with hundreds of variables